modules/random_cat_fact.py

```diff
@@ -0,0 +1,11 @@
+import requests
+import json
+
+def random_cat_facts():
+    try:
+        url = 'https://cat-fact.herokuapp.com/facts/random'
```

Comment:
TabError: inconsistent use of tabs and spaces in indentation
line 5 = space
line 6 = space
line 7 = tab
line 8 = tab
line 9 = tab
line 10 = space
line 11 = tab